Error while running "$ PELIAS_CONFIG=<path_to_config_json> npm start"

I am running the command "$ PELIAS_CONFIG=<path_to_config_json> npm start" from openstreetmap directory. Getting this error.

wordster@vpc-dev-es-02:~/sanket/openstreetmap$ PELIAS_CONFIG=pelias.json npm start

> pelias-openstreetmap@3.1.1 start /home/wordster/sanket/openstreetmap
> node index.js

/home/wordster/sanket/openstreetmap/stream/adminLookup.js:18
ipResolver = adminLookup.createWofPipResolver(config.imports.adminLookup.url);
                                                                        ^
TypeError: Cannot read property 'url' of undefined
    at Object.createStream [as adminLookup] (/home/wordster/sanket/openstreetmap/stream/adminLookup.js:18:82)
    at Object.streams.import (/home/wordster/sanket/openstreetmap/index.js:37:20)
    at Object.<anonymous> (/home/wordster/sanket/openstreetmap/index.js:50:17)
    at Module._compile (module.js:460:26)
    at Object.Module._extensions..js (module.js:478:10)
    at Module.load (module.js:355:32)
    at Function.Module._load (module.js:310:12)
    at Function.Module.runMain (module.js:501:10)
    at startup (node.js:129:16)
    at node.js:814:3

npm ERR! Linux 3.13.0-29-generic
npm ERR! argv "node" "/usr/local/bin/npm" "start"
npm ERR! node v0.12.7
npm ERR! npm  v2.14.2
npm ERR! code ELIFECYCLE
npm ERR! pelias-openstreetmap@3.1.1 start: `node index.js`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the pelias-openstreetmap@3.1.1 start script 'node index.js'.
npm ERR! This is most likely a problem with the pelias-openstreetmap package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     node index.js
npm ERR! You can get their info via:
npm ERR!     npm owner ls pelias-openstreetmap
npm ERR! There is likely additional logging output above.

npm ERR! Please include the following file with any support request:
npm ERR!     /home/wordster/sanket/openstreetmap/npm-debug.log

--- a/stream/adminLookup.js
+++ b/stream/adminLookup.js
@@ -2,6 +2,13 @@ var through = require('through2');
 var peliasConfig = require( 'pelias-config' ).generate();
 var wofAdminLookup = require('pelias-wof-admin-lookup');
 
+
+function sendPassthroughStream() {
+  return through.obj(function (doc, enc, next) {
+    next(null, doc);
+  });
+}
+
 /**
  * Generate a stream object that will handle the adminLookup when enabled.
  * When disabled, generate a passthrough stream.
@@ -14,15 +21,27 @@ function createStream(config, adminLookup) {
   config = config || peliasConfig;
   adminLookup = adminLookup || wofAdminLookup;
 
-  if (config.imports.openstreetmap.adminLookup) {
+  // disable adminLookup with empty config
+  if (!config.imports || !config.imports.openstreetmap) {
+    return sendPassthroughStream();
+  }
+
+  // admin lookup disabled
+  if (!config.imports.openstreetmap.adminLookup) {
+    return sendPassthroughStream();
+  }
+
+  // admin lookup enabled
+  if (config.imports.adminLookup && config.imports.adminLookup.url) {
     var pipResolver = adminLookup.createWofPipResolver(config.imports.adminLookup.url);
     return adminLookup.createLookupStream(pipResolver);
   }
-  else {
-    return through.obj(function (doc, enc, next) {
-      next(null, doc);
-    });
-  }
+
+  // throw error if flag is set but no URL configured
+  var message = 'Admin lookup is enabled but no url is specified in your config ' +
+    'at imports.adminLookup.url. Make sure a URL pointing to a Who\'s on First point ' +
+    'in polygon server (https://github.com/whosonfirst/go-whosonfirst-pip) is set in your config';
+  throw new Error(message);
 }
 
 module.exports = createStream;

